### PR TITLE
Follow up on data import

### PR DESCRIPTION
**Pull request by roschaefer**

### CODE REVIEW

File: src/middleware/fixImageUrlsMiddleware.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const legacyUrls = [
 
 export const fixUrl = (url) => {
   legacyUrls.forEach((legacyUrl) => {
-    url = url.replace(legacyUrl, '')
+    url = url.replace(legacyUrl, '/api')
```
Comment (roschaefer) [inline src/middleware/fixImageUrlsMiddleware.js]: @appinteractive probably not the best way to fix this. The backend *knows* frontend implementation details. It should be `''` not `'/api'` and the frontend image tags in the template needs to be changed, right?

Comment (appinteractive) [reply src/middleware/fixImageUrlsMiddleware.js]: Yeah we should change that in the Frontend to prefix them with /api

Comment (appinteractive) [reply src/middleware/fixImageUrlsMiddleware.js]: @robert?

Comment (roschaefer) [reply src/middleware/fixImageUrlsMiddleware.js]: Uhm, we can merge that and fix it later? It requires a change in the fronted

Comment (roschaefer) [reply src/middleware/fixImageUrlsMiddleware.js]: I just wanted to make we are on the same page here

**Pull request merged**